src/jaxsim/exceptions.py

```diff
@@ -0,0 +1,55 @@
+import jax
+
+
+def raise_if(
+    condition: bool | jax.Array, exception: type, msg: str, *args, **kwargs
+) -> None:
+    """
+    Raise a host-side exception if a condition is met. Useful in jit-compiled functions.
+
+    Args:
+        condition:
+            The boolean condition of the evaluated expression that triggers
+            the exception during runtime.
+        exception: The type of exception to raise.
+        msg:
+            The message to display when the exception is raised. It can be a fmt string,
+            and users can pass additional arguments to format the string.
+    """
+
+    # Check early that the fmt string is well-formed.
```

Comment:
```suggestion
            The message to display when the exception is raised. It can be a format string,
            and users can pass additional arguments to format the string.
    """

    # Check early that the format string is well-formed.
```